{"name": "输入错误URL签到失败", "status": "passed", "attachments": [{"name": "stdout", "source": "fc9f2497-0d35-41c5-8a47-c50e1fe38a7e-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "Parameters", "value": "{'casename': '输入错误URL签到失败', 'url': 'http://18.118.13.94:81/jeecg-boot/sys/api/users/signttt', 'data': '无', 'content_type': 'application/x-www-form-urlencoded', 'result': 'No message available', 'assert_way': 'message', 'mode': '签到'}"}], "start": 1648892528008, "stop": 1648892528803, "uuid": "3aa664da-5b41-4f32-b0bd-6ca086dc68bd", "historyId": "190e4828d4b90cff8c9c940841dc4326", "testCaseId": "e70cbdc81e516efc08fb54966aba8bee", "fullName": "main.testmain.TestMain#test_sign", "labels": [{"name": "story", "value": "签到"}, {"name": "epic", "value": "O&Kmarts 前端接口测试用例"}, {"name": "feature", "value": "签到"}, {"name": "parentSuite", "value": "main"}, {"name": "suite", "value": "testmain"}, {"name": "subSuite", "value": "TestMain"}, {"name": "host", "value": "DESKTOP-NB3FDE0"}, {"name": "thread", "value": "10872-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "main.testmain"}]}